app/navigation/AppNavigator.js

```
@@ -0,0 +1,29 @@
+import React from 'react';
+import { NavigationContainer } from '@react-navigation/native';
+
+import React from 'react';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'React'. Consider to delete it or rename it.

'React' is defined here:


(lint/suspicious/noRedeclare)